lemur/plugins/lemur_aws/sts.py

```diff
@@ -47,17 +47,57 @@ def assume_service(account_number, service, region='us-east-1'):
             security_token=role.credentials.session_token)
 
 
+def construct_role_arn(role_type='role', **kwargs):
+    account_number = kwargs.get('account_number') or \
+        current_app.config.get('AWS_DEFAULT_ACCOUNT_NUMBER') or \
+        boto3.client('sts').get_caller_identity().get('Account')
+    arn_data = {
+        'num': account_number,
+        'role_type': role_type,
+        'name': current_app.config.get('LEMUR_INSTANCE_PROFILE', 'Lemur')
+    }
+    arn_data['service'] = 'sts' if role_type == 'assumed-role' else 'iam'
+
+    return 'arn:aws:{service}::{num}:{role_type}/{name}'.format(**arn_data)
+
+
+def already_assumed_target_role(**kwargs):
```

Comment:
Another option to make the test happy, and like could be correct depending on the situation, would just be
```python
if 'account_number' in kwargs:
    return False
```